config/rbac/role_binding.yaml

```diff
@@ -1,7 +1,9 @@
----
```

Comment:
shouldn't all yaml files start with that?

yet some have it and some don't

Reply:
optionally, will restore it